templates/sovereign-rollup/run-sovereign-setup.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 # shellcheck disable=SC2034,SC2086
 
+# Fund L1 OP addresses.
+# 0xD3F2c5AFb2D76f5579F326b0cD7DA5F5a4126c35 is the OP Batcher Address on L1
+# The private key is an L1 prefunded address
+cast send \
+    --private-key bcdf20249abf0ed6d944c0288fad489e33f66b3960d9e6229c1cd214ed3bbe31 \
+    --rpc-url http://el-1-geth-lighthouse:8545 \
+    --value 100ether \
```

Comment:
Maybe this should be configured:
https://github.com/0xPolygon/kurtosis-cdk/blob/85f3b813ed90aafdc6b07a76949ac0eaa0a97b94/templates/contract-deploy/run-l2-contract-setup.sh#L45C18-L45C40